doozerlib/rpmcfg.py

```diff
@@ -357,36 +364,53 @@ def _build_rpm(self, scratch, record, terminate_event, dry_run=False):
             out_lines = out.splitlines()
 
             # Look for a line like: "Created task: 13949050" . Extract the identifier.
-            task_id = next((created_line.split(":")[1]).strip() for created_line in out_lines if
-                           created_line.startswith("Created task:"))
+            primary_task_id = int(next((created_line.split(":")[1]).strip() for created_line in out_lines if created_line.startswith("Created task:")))
 
-            record["task_id"] = task_id
+            record["task_id"] = primary_task_id
 
             # Look for a line like: "Task info: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=13948942"
             task_url = next((info_line.split(":", 1)[1]).strip() for info_line in out_lines if
                             info_line.startswith("Task info:"))
 
-            self.logger.info("Build running: {} - {}".format(self.rpm_name, task_url))
+            self.logger.info("Build running: %s - %s - %s", self.rpm_name, self.targets[0], task_url)
 
             record["task_url"] = task_url
 
-            # Now that we have the basics about the task, wait for it to complete
-            error = watch_task(self.runtime.group_config.urls.brewhub, self.logger.info, task_id, terminate_event)
+            task_ids = [primary_task_id]
+            # Build against secondary targets
+            secondary_targets = self.targets[1:]
+            if secondary_targets:
+                with self.runtime.shared_koji_client_session() as kcs:
+                    kcs.krb_login()
+                    source, _, _ = kcs.getTaskRequest(primary_task_id)
+                    for target in secondary_targets:
+                        self.logger.info(f"Submitting task for secondary target {target}")
+                        tid = kcs.build(source, target)
+                        task_ids.append(tid)
+                        task_url = "https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=" + str(tid)
+                        self.logger.info("Build running: %s - %s - %s", self.rpm_name, target, task_url)
+            record["targets"] = self.targets
+            record["task_ids"] = task_ids
```

Comment:
hmm... did you actually try this with a db configured? i'm not sure simpledb accepts an array as a value, and i really don't expect mariadb does. perhaps they already auto-convert to strings if they see this but... may be wise to do it explicitly here?

i mean, ideally we'd get as many records as builds out of this, but i can see that would be tricky to wedge into this change without a significant refactor.

Reply:
Haven't tried this with DB. I'll take a look (or convert it to a json string if it takes time)

Reply:
@sosiouxme I tested and revisited the simpledb code in Doozer. Currently Doozer only saves image build info into simpledb (see ImageDistGitRepo.update_build_db). This is only about rpms so it doesn't break.

Reply:
mkay. we'll probably want to start recording those before too long, but i suppose it's ok for now.